### PR TITLE
Chronograf RDD

### PR DESCRIPTION
**Pull request by tonerdo**

### CODE REVIEW

File: modules/chronograf-server/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Chronograf Server
```
Comment (brikis98) [inline modules/chronograf-server/README.md]: Will it be one server or a cluster of servers (ASG)?

File: modules/chronograf-server/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  script to configure and run Chronograf as either a meta node or a data node. 
+
+You can find the other parameters in [variables.tf](variables.tf).
+
+Check out the [examples folder](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples) for 
+fully-working sample code.
+
+## How do you connect to the Chronograf server?
+
+Once deployed you an simply access the web UI by visiting the public IP of the server on the specified port.
```
Comment (brikis98) [inline modules/chronograf-server/README.md]: Will this module output this IP? If so, mention that and the name of the output variable. But if this is an ASG, then you'd need a load balancer...

File: modules/chronograf-server/README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+## What's included in this module?
+
+This module creates the following:
+
+* [Auto Scaling Group](#auto-scaling-group)
+* [Security Group](#security-group)
```
Comment (brikis98) [inline modules/chronograf-server/README.md]: This outline doesn't correspond to the items below

File: modules/chronograf-server/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+### Security Group
+
+The EC2 Instance has a Security Group that allows minimal connectivity:
+
+* All outbound requests
+* Inbound SSH access from the CIDR blocks and security groups you specify
```
Comment (brikis98) [inline modules/chronograf-server/README.md]: What about inbound access to the UI itself? On what port?

File: modules/chronograf-server/README.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+If you wish to use dedicated instances, you can set the `tenancy` parameter to `"dedicated"` in this module. 
+
+### Encryption
+
+This module does not currently support specifying encryption information. The official [documentation](
+https://docs.influxdata.com/chronograf/v1.7/administration/managing-security/#configuring-tls-transport-layer-security-and-https) contains a guide for enabling SSL.
+
+### Security groups
+
+This module attaches a security group to the EC2 Instance that allows inbound requests as follows:
```
Comment (brikis98) [inline modules/chronograf-server/README.md]: What about the port for accessing the UI?

File: modules/chronograf-server/README.md (hunk 1 above)
Comment (tonerdo) [reply modules/chronograf-server/README.md]: One server

File: modules/chronograf-server/README.md (hunk 4 above)
Comment (tonerdo) [reply modules/chronograf-server/README.md]: This module doesn't handle that. It happens in the `chronograf-security-group-rules` module

File: modules/chronograf-server/README.md (hunk 3 above)
Comment (tonerdo) [reply modules/chronograf-server/README.md]: Fixed

File: modules/chronograf-server/README.md (hunk 5 above)
Comment (tonerdo) [reply modules/chronograf-server/README.md]: Same as above ^

**Pull request merged**